cds_dojson/marc21/fields/books/base.py

```diff
@@ -52,6 +52,11 @@ def acquisition_source(self, key, value):
         _acquisition_source.update(
             {'method': mapping(ACQUISITION_METHOD,
                                clean_val('s', value, str))})
+        date = clean_val('w', value, int, regex_format=r'\d{4}$')
```

Comment:
Sorry, silly question, but here we expect 4 digits only? Or at least 4 digits?
Are we sure that than we are able to split year, week?

Reply:
we expect 4 digits, everything else should fail as it means the value has wrong format and it should be fixed by the librarian